liquibase-couchbase/src/test/java/liquibase/ext/couchbase/executor/CouchbaseExecutorTest.java

```diff
@@ -89,4 +97,34 @@ void Should_wrap_exception() {
         assertThatExceptionOfType(StatementExecutionException.class)
                 .isThrownBy(() -> couchbaseExecutor.execute(couchbaseStatement));
     }
+
+    @Test
+    void Should_send_comment_to_log() {
+        try (MockedStatic<Scope> mockedStatic = Mockito.mockStatic(Scope.class)) {
+            String comment = "comment";
+            Scope scope = mock(Scope.class);
```

Comment:
`currentScope`? Because it's not obvious what's happening with `mockedStatic` and `scope`

Reply:
done